### ISSUE TITLE
PoE2 support

### ISSUE THREAD

**Issue by TheOne320** [ghostscript3r/poe-trade-official-site-enhancer#47]
I still like this trade site enhancer most of all. Will you be updating it for PoE2 and fixing the bugs it currently has?

**Comment by agaragnani** [comment]
Yes, PoE2 revamped my interest for this project too, so I also finally finished some new features that were cooking for a very long time... check out the new version

**Comment by ghostscript3r** [comment]
Yes, PoE2 revamped my interest for this project too, and I also finally finished some new features that were cooking for a very long time...
